### PR TITLE
Moved centralfcu from historical to current shared credentials

### PR DESCRIPTION
**Pull request by ezfe**
### Overall Checklist
- [x] I agree to the project's [Developer Certificate of Origin](https://github.com/apple/password-manager-resources/blob/main/DEVELOPER_CERTIFICATE_OF_ORIGIN.md)
- [x] The top-level JSON objects are sorted alphabetically
- [x] There are no [open pull requests](https://github.com/apple/password-manager-resources/pulls) for the same update

#### for shared-credentials.json
- [x] There's evidence the domains are currently related (SSL certificates, DNS entries, valid links between sites, legal documents etc.)
- [x] If using `shared`, the new group serves login pages on each of the included domains, and those login pages accept accounts from the others. (For example, we wouldn't use a `shared` association from `google.co.il` to `google.com`, because `google.co.il` redirects to `accounts.google.com` for sign in.)

Proof of shared credentials:

Entering a correct password on https://www.centralfcu.com goes directly to the account page. However, entering *incorrect* credentials on https://www.centralfcu.com loads a login page on https://www.centralfcu.org/tob/live/usp-core/app/initialLogin. This establishes a relationship from the `.com` to the `.org`.

Navigating to https://www.centralfcu.org and clicking the logo will redirect to the `.com` version. This establishes the reverse relationship.

### CODE REVIEW

**Comment by bradcush** [comment]
@ezfe Thanks for the submission! Just to clarify, is there a difference between logging in to `centralfcu.com` versus `centralfcu.org`, meaning the site that you have access to is actually different once you've gained access?

File: quirks/shared-credentials-historical.json [hunk 1]
```diff
@@ -153,12 +153,6 @@
             "asiamiles.com"
         ]
     },
-    {
-        "shared": [
```
Comment (bradcush) [inline quirks/shared-credentials-historical.json:157]: Specify using `from`/`to` keys, the `to` being `centralfcu.org`

**Comment by ezfe** [comment]
@rmondello Can you confirm whether `from`/`to` should be used when the login page appears on both domains, but always redirects *after* login to the second domain?

**Comment by rmondello** [comment]
> @rmondello Can you confirm whether `from`/`to` should be used when the login page appears on both domains, but always redirects _after_ login to the second domain?

That's not the meaning. It's for when a domain redirects to another domain to do its login.

**Comment by ezfe** [comment]
Thanks - I will resolve the conflict but then this is ready to merge

**Comment by ezfe** [comment]
@bradcush Please re-review

**Review by bradcush — APPROVED** [review]

**Comment by bradcush** [comment]
> Thanks - I will resolve the conflict but then this is ready to merge

Sorry for the confusion here, thanks for clarifying @rmondello!

**Pull request merged**